packages/components/tests/lib/controllers/selection-controller.test.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-env jest */
+import SelectionController from '@comp/lib/controllers/selection-controller.js'
+import UIEventController from '@comp/lib/controllers/ui-event-controller.js'
+import MouseDblClick from '@comp/lib/custom-pointer-events/mouse-dbl-click.js'
+
+describe('SelectionController', () => {
+  let selC
+
+  afterEach(() => {
+    jest.resetModules()
+  })
+
+  afterAll(() => {
+    jest.clearAllMocks()
+  })
+
+  it('UIController - constructor: should create an instance with default arguments', () => {
```

Comment:
I belive, that titles should start from SelectionController.
I defined the title format this way - to be able easily to identify what test case is failed, when a bunch of tests are executed.

Reply:
Oh, I've overlooked that completely. Thanks for catching this! Will rename.